packages/cloudformation/templates/givesource-api-deploy.yml

```diff
@@ -71,7 +71,7 @@ Resources:
           HttpMethod: GET
           LoggingLevel: INFO
           DataTraceEnabled: true
-          CachingEnabled: false
+          CachingEnabled: true
```

Comment:
I believe this was intentionally set to false because we believed it was leading to nonprofits getting cached donations reports for other nonprofits. I'm not sure if we had evidence of that or if it was just a guess. I'm hesitant to turn caching back on unless we have a really good reason to. Was there a specific performance benefit this gave us?